src/handlers/Handler.ts

```diff
@@ -344,7 +350,9 @@ export class Handler implements IHandler {
                 this.sender.id,
             );
 
+            this.app.getLogger().info('🔥 Storing room ID in Config handler:', this.room.id);
```

Comment:
remove emojis , and only add logs for thing that is okay to be disclosed to end user not everything .